cmd/test-configs/irene.toml

```diff
@@ -2,18 +2,18 @@
 
 usedurablestore = true
 
+guiport = 5006
 msgport = 3006
 rpcport = 4006
-guiport = 5006
 
 # PeerID: 16Uiu2HAmHntR3SGeS7iF2tdeNBefSahXBhmTrqVozVLHydxzkaZn
 # SCAddr: 0x111A00868581f73AB42FEEF67D235Ca09ca1E8db
 pk = "febb3b74b0b52d0976f6571d555f4ac8b91c308dfa25c7b58d1e6a7c3f50c781"
 
 chainpk = "7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6"
 
-chainurl = "ws://127.0.0.1:8545"
 chainauthtoken = ""
+chainurl = "ws://127.0.0.1:8545"
 
-bootpeers = ""
-usemdns = true
+# bootpeer = Ivan
+bootpeers = "/ip4/127.0.0.1/tcp/3008/p2p/16Uiu2HAm1hgN2MkrhGen8JPrBBYyXACbZtfqJmraN53XiHYCeoFi"
```

Comment:
Previously Irene was no one's boot peer and didn't have a boot peer to connect to; so she was left out. Poor Irene 😢